pkg/otlp/model/go.mod

```diff
@@ -6,15 +6,15 @@ replace github.com/DataDog/datadog-agent/pkg/quantile => ../../quantile
 
 require (
 	github.com/DataDog/datadog-agent/pkg/quantile v0.37.0-rc.3
+	github.com/DataDog/sketches-go v1.4.1
 	github.com/patrickmn/go-cache v2.1.0+incompatible
 	github.com/stretchr/testify v1.7.1
-	go.opentelemetry.io/collector/pdata v0.50.0
-	go.opentelemetry.io/collector/semconv v0.50.0
+	go.opentelemetry.io/collector/pdata v0.51.0
+	go.opentelemetry.io/collector/semconv v0.51.0
```

Comment:
Is this bump required? It's not a good idea to have `pdata` at v0.51 but `go.opentelemetry.io/collector` at v0.50.0 on the main go.mod. I would rather bump everything to v0.51.0 if it's easy for you